templates/member/admin/memberip.html

```diff
@@ -37,4 +37,24 @@
             {% endfor %}
         </ul>
     </div>
+
+    {% if ":" in ip %} <!--Checks if it's an IPV6 to show the members from the same IPV6 network-->
```

Comment:
Pour commenter le code du _template_, il vaut mieux utiliser des commentaires du moteur de gabarits de Django (sous la forme `{# commentaire #}`) que des commentaires HTML. En effet, de tels commentaires ne sont pas inclut dans le rendu HTML. Ce commentaire l'est, et d'une c'est inutile, de deux ça ne ferait aucun sens en lisant le code HTML car la condition ne serait plus là.

De manière générale, on n'utilise les commentaires HTML que si _vraiment_ on veut que le commentaire soit visible dans le code source généré de la page. Dans les faits, c'est très rare : généralement on ne veut qu'annoter pour nous, mais conserver le commentaire et l'envoyer à tous les visiteurs n'est que peu utile. Ça peut servir pour des mentions d'attribution, par exemple, mais sinon…
```suggestion
    {# Checks if it's an IPV6 to show the members from the same IPV6 network #}
    {% if ":" in ip %}
```

Reply:
C’est fait maintenant :)